xilem_web/src/lib.rs

```diff
@@ -53,6 +54,7 @@ pub use attribute_value::{AttributeValue, IntoAttributeValue};
 pub use class::{AsClassIter, Class, Classes, ElementWithClasses, WithClasses};
 pub use context::ViewCtx;
 pub use element_props::ElementProps;
+pub use memoized_await::{memoized_await, MemoizedAwait};
```

Comment:
I'm still not sure about the actual namespacing in xilem_web (and I think this is certainly also relevant for xilem), this exports `MemoizedAwait` which is probably nice for explicitely naming types, though I doubt that will likely happen (mostly because of the countless generic params necessary), So I'm not sure whether that type should even be exported, as it pollutes the namespace, but I slightly lean to exporting it.

I'm not sure, whether I should just create a new `mod` `view` and put all things that implement a `View` + things relevant for those into it, that means also `elements`, but I could also imagine just namespacing views in the root, e.g. with this view in `xilem_web::concurrent`...

Reply:
Continuing loud thoughts: I've moved the view now in `xilem_web::concurrent`, as I think that this cleans it up a little bit and is more future proof.